Source/CommandLine.cpp

```diff
@@ -462,10 +462,18 @@ bool shouldPerformCommandLine (const String& commandLine)
 //==============================================================================
 std::pair<juce::String, PluginTests::Options> parseCommandLine (const juce::ArgumentList& args)
 {
+    // prioritize explicit arguments
     auto fileOrID = args.getValueForOption ("--validate");
 
+    // but let user lazily supply the plugin path as the last arg
+    if (fileOrID.isEmpty())
```

Comment:
I don't think this should be needed as the `ArgumentList` supplied should already be formatted to add the `--validate` option before an implicit plugin path for the last arg?

Reply:
This confused me as well, but I assumed there was some reason the existing logic using `args.arguments.getLast()`...

Just took a deeper look and it's because [`getValueForOption` only returns a value when it matches an equal sign for long arguments](https://github.com/juce-framework/JUCE/blob/master/modules/juce_core/misc/juce_ConsoleApplication.cpp#L97-L108). So, the string would have to be `--validate=some/plugin/path` to work correctly. It seems to always be empty here otherwise.

This sorta motivates me more to look into CLI11 down the line... the JUCE option parsing seems a bit crusty/hard to use and the `getLast` call is likely also [why this bug exists](https://forum.juce.com/t/anouncing-pluginval-an-open-source-cross-platform-plugin-validation-tool/27710/191?u=sudara).

Seeing if i can find a solution...

Reply:
Ok, so it seems to be a lose-lose situation unless we embrace the fact that JUCE wants to see `--validate=/some/path`. 

I updated the PR with how I would resolve this (rewriting the implicit and explicit validate case).

This also resolves the order independence bug, which I added a test for.